var/spack/repos/builtin/packages/pfunit/package.py

```diff
@@ -40,16 +37,32 @@ class Pfunit(CMakePackage):
 
     depends_on('python@2.7:', type=('build', 'run'))  # python3 too!
     depends_on('mpi', when='+mpi')
+    depends_on('m4', when='@4.1.5:')
```

Comment:
This is usually a build dependency:
```suggestion
    depends_on('m4', when='@4.1.5:', type='build')
```